src/Converter.php

```diff
@@ -210,7 +227,16 @@ function ($match) use ($replace) {
                         $this->components[$match['given']] = preg_replace('/\{tailwindo\|([^\}]+)\}/', '$1', $replace);
                     }
 
-                    return $replace;
+                    return $this->prefix ? trim(implode(' ', array_filter(array_map(function ($class) {
```

Comment:
Use `if` conditions and multiple lines for readability. multiple functions in one line makes debugging harder.